packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -105,15 +104,68 @@ + (NSString * _Nullable)version {
   return nil;
 }
 
+// Expo developers: Enable the below code by running
+//     export EX_DEV_LAUNCHER_URL=http://localhost:8090
+// before doing pod install. This will cause the controller to see if
+// the expo-launcher packager is running, and if so, use that instead of
+// the prebuild bundle.
+// See the post_install step in expo/scripts/autolinking.rb
+
+- (NSURL *)devLauncherURL
```

Comment:
I think this should be marked as `Nullable` - we hit a bug not too long ago with this type annotation - Swift was not able to figure out it when it was `nil` without the `Nullable` flag and would crash

Reply:
albeit this is a private method for now just something to be aware of

Reply:
Good catch, and I made this change